mypy/test/teststubtest.py

```diff
@@ -635,7 +635,8 @@ def f5(__b: str) -> str: ...
         )
         yield Case(
             stub="""
-            from typing import deprecated, final
+            from typing import final
+            from warnings import deprecated
```

Comment:
I think it recognised `typing.deprecated` because Shantanu added it to stubtest's typing stub here:

https://github.com/python/mypy/blob/086bb5116ef3eb8987e8ff46a2172746b2569afb/mypy/test/teststubtest.py#L75

But I don't think we have `@deprecated` in any of mypy's fixtures for the `warnings` module yet

Reply:
Ah right. I should actually test this the next time before I change something.

Reply:
True, we don't really get anything out of putting it in warnings.

Reply:
Changed in 804099902bdb5a378b34cc150bdd4e74f9324481